extensions/multirecordings.sigmf-ext.md

```diff
@@ -0,0 +1,103 @@
+recording, they would bloat and complicate the use of that recording.
+Furthermore, it would be difficult to process any aspect of the data without
+parsing all of the recorded data. This extension enables you to treat such
+fields as *simply another `SigMF Recording`*.
+
+For recordings that digitize multiple streams of information (e.g., a
+phased-array receiver with multiple channels), this extension enables you to
+capture the multiple streams and link them such that each one can be treated as
+a separate `SigMF Recording` or they may be treated together as one.
+
+Per the SigMF specification, all recordings must be independently evaluated and
+be able to stand-alone.
+
+## Datatypes
+
+| type      | long-form name        | description                                             |
+| --------- | --------------------- | ------------------------------------------------------- |
+| recording | string recording name | A string that indicates the name of a `SigMF Recording` |
+
+The `recording` datatype is the primary mechanism by which SigMF Recordings are
+linked. This datatype is the name of a SigMF Recording, **without** extensions
+(i.e., given a recording comprising `N.sigmf-meta` and `N.sigmf-data`, the
+`recording` datatype value would simply be `N`).
+
+The metadata and dataset files of the recording MUST be in the local directory
+(i.e., local to the SigMF Recording in which they are referenced, and thus in
+the same SigMF Archive).
+
+### `core` Namespace Support
+
+If the `multirecordings` extension is supported, it MAY be used in the following
+name/value pairs within the `core` namespace.
+
+#### Capture Segment Objects
+
+The following are existing name/value pairs in capture segment objects in the
+`core` namespace:
+
+| name           | required | type      | description  |
+| -------------- | -------- | --------- | ------------ |
+| `frequency`    | false    | recording | The center frequency of the signal in Hz.|
+
+### `multirecordings` Namespace
+
+This extension adds the following name/value pairs under the `multirecordings`
+namespace.
+
+#### Global Object
+
+| name          | required | type      | description                         |
+| ------------- | -------- | --------- | ----------------------------------- |
+| `master`      | false    | recording | The primary recording this recording is linked to.|
+
+The `multirecordings:master` field in the `global` object allows the user to
+indicate another SigMF recording that acts as the 'master' or 'parent' of this
+one, effectively providing a mechanism for bi-directional link between SigMF recordings.
+
+#### Capture Segment Objects
+
+| name           | required | type      | description  |
+| -------------- | -------- | --------- | ------------ |
+| `streams`      | false    | object    | List of SigMF recordings that represent multiple streams.|
+
+The `multirecordings:streams` field in a capture segment object is a JSON array
```

Comment:
How does this work if there are multiple segments at nonzero offsets? What does that imply for channels other than 'this' channel?

Reply:
@jacobagilbert - I don't think I understand the gap you're pointing out. Can you explain it in a bit more detail for me?

Reply:
Sure - I think my question here has to do with this `streams` object being part of a `captures` segment, which starts at a user-defined sample (not necessarily 0) and extends until the next segment or EOF. That seems like something that might be a challenge. It also might be well thought out, so this is not to say 'this is a bad idea' -- rather 'is this what was intended'? Or perhaps this should be in `global`?

Reply:
I removed this from my re-write of the extension, and then realized why it was there 😅

This is needed to point from the primary recording down to children. That said, I agree with your comment about it being in captures. The new PR moves this to global.